tests/go/GoTypeBearTest.py

```diff
@@ -0,0 +1,38 @@
+from bears.go.GoTypeBear import GoTypeBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = """package main
```

Comment:
as i've seen, as a codestyle, we don't use anything else on the same line as the triple backquotes. same applies to the bad_file as well :)